core/repository/consensus/merkle_transaction_repository.hpp

```diff
@@ -5,12 +5,11 @@
 #include <memory>
 #include "../../model/transactions/abstract_transaction.hpp"
 
-#include "../../consensus/consensus_event.hpp"
-
 namespace merkle_transaction_repository {
-//bool commit(std::string const hash, std::unique_ptr<consensus_event::ConsensusEvent> const tx);
-bool commit(std::string, const std::unique_ptr<consensus_event::ConsensusEvent> &event);
-std::unique_ptr<abstract_transaction::AbstractTransaction> findLeaf(std::string const hash);
+
+bool commit(std::string, const std::unique_ptr<abstract_transaction::AbstractTransaction> &event);
+
+bool leafExists(std::string const hash);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Function parameter 'hash' should be passed by reference.](https://www.codacy.com/app/soramitsu/iroha/file/4278611608/issues/source?bid=3712501&fileBranchId=3837537#l12)